Resources/Prototypes/ai_factions.yml

```
@@ -47,6 +93,9 @@
 
 - type: npcFaction
   id: SimpleHostile
+  friendly:
+  - AnimalFriend
+  - SalvageMob
```

Add explanatory comment
```suggestion
  friendly:
  - AnimalFriend # DeltaV - Add AnimalFriend faction
  - SalvageMob
```

Also explain why SimpleHostile is now friendly to SalvageMob?

---

So that they dont attack eachother, Salv mob is only goliath, hivelord, hivelordbrood, and basilisk. Req by direction to seperate the two so that Ovinia arent supersalvager